buildpipeline/DotNet-Trusted-Publish-Symbols.json

```diff
@@ -1,5 +1,24 @@
 {
   "build": [
+    {
+      "enabled": true,
+      "continueOnError": false,
+      "alwaysRun": false,
+      "displayName": "Run script $(VS140COMNTOOLS)\\VsDevCmd.bat",
+      "timeoutInMinutes": 0,
+      "task": {
+        "id": "bfc8bf76-e7ac-4a8c-9a55-a944a9f632fd",
+        "versionSpec": "1.*",
+        "definitionType": "task"
+      },
+      "inputs": {
+        "filename": "$(VS140COMNTOOLS)\\VsDevCmd.bat",
```

Comment:
Note this is likely going to cause us problems in the future as we move to VS2017 only. We may need to use a more flexible script in finding the correct vsdevcmd to call or not use .NET Framework msbuild.

Reply:
I just copied this from DotNet-Trusted-Publish.json (and it is also in DotNet-Trusted-Tests-Publish.json).  It looks like the coreclr publish uses the follow step.  Will that work in corefx?

    {
      "environment": {},
      "enabled": true,
      "continueOnError": false,
      "alwaysRun": false,
      "displayName": "Setup vs dev env",
      "timeoutInMinutes": 0,
      "task": {
        "id": "bfc8bf76-e7ac-4a8c-9a55-a944a9f632fd",
        "versionSpec": "1.*",
        "definitionType": "task"
      },
      "inputs": {
        "filename": "$(Pipeline.SourcesDirectory)\\setup_vs_tools.cmd",
        "arguments": "",
        "modifyEnvironment": "true",
        "workingFolder": "",
        "failOnStandardError": "false"
      }
    },

Reply:
It will not currently work as corefx doesn't have the setup_vs_tools.cmd script yet but I do think we will want something like it eventually. This isn't a blocker for your work just something I noticed thus called out.